consensus/spos/bls/subroundEndRound.go

```diff
@@ -417,8 +418,7 @@ func (sr *subroundEndRound) doEndRoundJobByLeader() bool {
 	}
 
 	if sr.EnableEpochsHandler().IsFlagEnabledInEpoch(common.EquivalentMessagesFlag, sr.Header.GetEpoch()) {
-		sr.worker.SetValidEquivalentProof(sr.GetData(), proof)
-		sr.Blockchain().SetCurrentHeaderProof(proof)
+		sr.worker.SetValidEquivalentProof(proof)
```

Comment:
should be saved into equivalentProofsTracker

Reply:
done